script.js

```diff
@@ -1,10 +1,14 @@
 /* eslint-disable linebreak-style */
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-len */
+/* eslint-disable linebreak-style */
```

Comment:
[OPTIONAL] Please don't disable linters. Linters are there to help you to write standardized code.